brainscore_language/plugins/fedorenko2016/data_packaging.py

```diff
@@ -0,0 +1,114 @@
+from glob import glob
+
+import logging
+import numpy as np
+import os
+import scipy.io
+import scipy.stats
+import sys
+from pathlib import Path
+
+from brainio.assemblies import NeuroidAssembly
+from brainscore_language.utils.s3 import upload_data_assembly
+
+_logger = logging.getLogger(__name__)
+
+"""
+The code in this package was run only once to initially upload the data, and is kept here for reference.
+"""
+
+
+def upload_fedorenko2016():
+    assembly = load_fedorenko2016()
+    upload_data_assembly(assembly,
+                         assembly_identifier="Fedorenko2016.language",
+                         bucket_name="brainscore-language")
+
+
+# adapted from
+# https://github.com/mschrimpf/neural-nlp/blob/cedac1f868c8081ce6754ef0c13895ce8bc32efc/neural_nlp/neural_data/ecog.py#L18
+
+def load_fedorenko2016():
```

Comment:
maybe we need to also specify that this is the averaged version, since Greta and I will soon follow up with a time-expanded version of this dataset, so how should we disambiguate between multiple such versions of a dataset?

Reply:
Fedorenko2016mean?

Reply:
up to you, feel free to change the naming as you see fit (e.g. send in a PR after this one is merged to modify?)